deploy/rpm/SPECS/karavi-authorization.spec

```diff
@@ -14,7 +14,7 @@
 
 Name:           karavi-authorization
 Version:        1.5
-Release:        0
+Release:        1
```

Comment:
@sharmilarama does this need to be updated? Previous value is zero meaning we haven't updated it before.

Reply:
@xuluna Did you build the rpm using this version update? Does it build with 1.5.1 as the version - `karavi-authorization-1.5-1.x86_64.rpm`?

Reply:
@sharmilarama Yes I tested it e2e and the build created is karavi-authorization-1.5-1.x86_64.rpm
```
$ ls deploy/rpm/x86_64
karavi-authorization-1.5-1.x86_64.rpm
```